lib/vector-api-client/graphql/schema.json

```diff
@@ -1008,34 +1008,34 @@
         {
           "kind": "OBJECT",
           "name": "EventNotification",
-          "description": "A notification regarding events observation",
+          "description": null,
```

Comment:
Nit: I think this may have been nullified due to a larger doc comment.

For structs that are implemented as GraphQL types, it may be worth adding engineering context using standard comments, and reserving doc comments for that which we want to surface to API clients.

No action required in this case; just an FYI.

Reply:
Gotcha, I'd shifted this description over to `Notification` and `EventNotification` slipped my mind. Happy to fix this up.